LeProHQpy/src/LeProHQ/cq1_.py

```diff
@@ -36,7 +36,17 @@ def cq1t(proj, cc, xi, eta):
         * (a11 * np.log(beta) + a10)
     )
 
+def cq1hv(proj,cc,xi,eta):
+    """High virtuality limit of cq1."""
+    l = np.log(xi)
+    z = xi / (4.*(1.+eta) + xi)
+    n = xi * 4**2 * np.pi / z
+    if "F2" == proj:
```

Comment:
It is customary to write it in the inverse fashion:
```suggestion
    if proj == "F2":
```

Reply:
I did fall back to C style - reverted in [ab52f96](https://github.com/felixhekhorn/LeProHQ/pull/2/commits/ab52f967cffa0af30fb3793a33a12c12a47e6540)